packages/react-scripts/config/webpack.config.js

```diff
@@ -47,9 +47,10 @@ const FilterWarningsPlugin = require('webpack-filter-warnings-plugin');
 const {
   BanBackendImportsPlugin,
   IModeljsLibraryExportsPlugin,
-  CopyBentleyStaticResourcesPlugin,
```

Comment:
Any reason to not update the `CopyBentleyStaticResourcesPlugin` directly instead of redefining here? I think we'd want to make similar changes to the backend webpack config we have which also uses this plugin.

Reply:
1. Mostly because I didn't want to wait for a new release of imjs to get this resolved.
2. Would we be importing and using the `copy-webpack-plugin` as a wrapper in the current `CopyBentleyStaticResourcesPlugin` or duplicate the code? I was hoping to just eliminate the plugin we had. 
3.  I wasn't aware backends needed any assets. Looks like they copy to `assets` dir in the `BackendDefaultPlugin` where as here we copy to the `public` dir, and i guess we can parameterize our plugin to handle both, but i figure its just easier to use theirs.

Reply:
I think it'd make sense to take a dependency on the `copy-webpack-plugin` in `@bentley/webpack-tools-core` and use it for both approaches. Then parameterize it the folder name to copy.

The main reason is we'll need to add the `@itwin` support for backends anyway so either we copy the stuff you did here with `copy-webpack-plugin` to that config too or we just switch around the existing plugin.

cc: @wgoehrig for his thoughts on this.

Reply:
Ok that make sense, also wasn't sure about needing assets from `itwin` scoped pkgs in the backend. I should really look into what those assets are. 

But yea sounds good, lets wait on Bill and see what he wants to do.

Reply:
Ended up moving plugin into core, and opted to do localization in a separate plugin.